### ISSUE TITLE
can we open 2 or 3 or more pokebuddys in one pc at same time? 

### ISSUE THREAD

**Issue by yccccbb** [finndev/PokeBuddy#447]
Hi guys, this is my first time here, i wanna know can we do that .if we can, will we get some ip address problems? Sorry for my English, hope some one can tell me, thanks  (-^〇^-)

**Comment by oOBlueArrowOo** [comment]
havent tried

**Comment by oOBlueArrowOo** [comment]
u might if they are completely separate or different bots

**Comment by WilliamcaFox** [comment]
Yes, I advise using multiple folders though so you can have seperate configs for each bot. Set up how you want your pokemon handled then simply copy paste.

I currently have 6 running.

**Comment by yccccbb** [comment]
Wow, thank you guys, i will try it now, u guys have a good good day

**Comment by exclamati0n** [comment]
Only 6?  >=)

**Comment by oOBlueArrowOo** [comment]
im using one xD

**Comment by WilliamcaFox** [comment]
> Only 6? >=)

I was too lazy to set up more throwaway gmail accounts!

**Comment by oOBlueArrowOo** [comment]
do u get more xp?

**Comment by oOBlueArrowOo** [comment]
@WilliamcaFox is ur name william cabrera by any chance?

**Comment by Mothanick** [comment]
6 pokebuddys is that for one acc? or Multiples

**Comment by yccccbb** [comment]
I think is 6 accounts for 6bots, lol

**Comment by exclamati0n** [comment]
Multiple accounts I assume?

Why possibly "idiot beacon" yourself running 6 instances of the same account?

**Comment by oOBlueArrowOo** [comment]
so it doesn't do more exp if u have different bot running the same account

**Comment by keytonic** [comment]
dont go through the hassle of making fake gmail accounts just reg new ptc accounts and use https://www.mailinator.com/ for throw away emails
